src/reducers/entities/integrations.ts

```diff
@@ -42,6 +42,22 @@ function incomingHooks(state: IDMappedObjects<IncomingWebhook> = {}, action: Gen
 
         return state;
     }
+    case ChannelTypes.RECEIVED_CHANNEL_UNARCHIVED: {
```

Comment:
This block seems unnecessary. The webhooks for this channel can't be undeleted when the channel is undeleted. I assume whatever code loads the webhooks will be called again to reload these later on.